src/Analysis/GWASResultsV2/Views/Home/HomeTable/filterTableData.js

```diff
@@ -0,0 +1,58 @@
+import moment from 'moment';
+
+const filterBySearchTerm = (initData, key, searchTerm) => initData.filter((obj) => obj[key]
```

Comment:
Good idea. This has been revised in each of the filtering methods to use ```data``` as a variable name instead of ```initData```

```

const filterBySearchTerm = (data, key, searchTerm) => data.filter((obj) => obj[key]
  .toString()
  .toLowerCase()
  .includes(searchTerm.toLowerCase()),
);

const filterByJobStatuses = (data, jobStatusSelections) => data.filter((item) => jobStatusSelections.includes(item.phase));

const filterByDateRange = (data, key, dateSelection) => data.filter((obj) => {
  const utcDate = moment.utc(obj[key]);
  return (
    utcDate.isSameOrAfter(dateSelection[0])
      && utcDate.isSameOrBefore(dateSelection[1])
  );
});

const filterTableData = (data, homeTableState) => {
  let filteredDataResult = data;
  if (homeTableState.nameSearchTerm.length > 0) {
    filteredDataResult = filterBySearchTerm(
      filteredDataResult,
      'name',
      homeTableState.nameSearchTerm,
    );
  }

```